react/frontpage/components/shows.scss

```diff
@@ -0,0 +1,212 @@
+/* SHOWS CONTENT */
+
+.graphStyle {
+  width: 60%;
+  float: left;
+}
```

Comment:
We need to split up this file into individual files for individual components... We should make separate `ShowsBlock.scss` and `ShowsGraph.scss` files, for example

Is this to share styles? Any shared lines can be in the `/react/frontpage/common.scss` file, which you can import with SASS:

```
/**
EventsTab.scss, etc.
**/
@import '../common';
```